test/inductor/test_torchinductor_opinfo.py

```diff
@@ -319,6 +319,78 @@ def wrapper_noop_set_seed(op, *args, **kwargs):
 )
 
 
+def default_comparator_assert(
```

Comment:
Better to move this into `check_model` function and assign the `equal_check` to this default function if `equal_check` is `None`.